sdk/workloads/Azure.ResourceManager.Workloads/tests/SessionRecords/SviCRUDTests/TestSVICrudOperations()Async.json

```diff
@@ -0,0 +1,1386 @@
+{

```

Comment:
We only have the test records in asynchronous mode but not synchronous mode, I'd like to understand why we missed the synchronous test records. If it is because that we only run the tests with asynchronous methods, please [run it with synchronous methods](https://github.com/Yao725/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#writing-scenario-tests) again and record the tests so the CI can pass.

Reply:
Added all tests.